main/hdparm/APKBUILD

```diff
@@ -20,11 +24,10 @@ build() {
 
 package() {
 	cd "$builddir"
-	mkdir -p "$pkgdir"/usr "$pkgdir"/sbin
-	make DESTDIR="$pkgdir" install
-	install -m755 contrib/idectl "$pkgdir"/sbin
-	install -m755 contrib/ultrabayd "$pkgdir"/sbin
-	install -D -m 644 LICENSE.TXT $pkgdir/usr/share/licenses/hdparm/LICENSE.TXT
+	make DESTDIR="$pkgdir" sbindir=/usr/bin install
```

Comment:
I see that you move `/sbin/hdparm` to `/usr/bin/hdparm`. It would be nice with some sort of explanation why this is needed in the commit message. And if this is what we really want to do then we should also patch busybox hdparm to the same location.

Reply:
@ncopa Sure, source url restored.
Regarding the `/sbin/hdparm` to `/usr/bin/hdparm` move, i thought it is more consistent, because `hdparm` is not an integral part of the base system.
This of course can definitely be reverted.